pkg/qrcodegenerator/qrcode/gateways/ttigpro1.go

```diff
@@ -0,0 +1,140 @@
+// Copyright © 2024 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gateways
+
+import (
+	"regexp"
+	"strings"
+
+	"go.thethings.network/lorawan-stack/v3/pkg/ttnpb"
+	"go.thethings.network/lorawan-stack/v3/pkg/types"
+)
+
+const (
+	formatIDttigpro1 = "ttigpro1"
+
+	euiLength        = 16
+	OwnerTokenLength = 12
+)
+
+// TTIgpro1 is a format for gateway identification QR codes.
+type TTIgpro1 struct {
+	GtwEUI     types.EUI64 // lowercase base16, 16 chars long
```

Comment:
```suggestion
	GatewayEUI     types.EUI64
```

`types.EUI64` implies the type

Reply:
fixed